serverless/components
# Pass org to find an access key and use it - if one exists. Update rea…

…dme for proper SERVELRESS_ACCESS_KEY env var.

## What has been implemented?

Closes #627

## Steps to verify

- remove the ID token from your `~/.serverlessrc` file (but ensure you have an access key set for the org).
- You should still be able to deploy, run dev mode, deploy all, and run info.
- Sls registry will not work (org is not present in the yaml for a component template at this time)

## Todos:
